### PR TITLE
Add recipe for graph-pes

### PR DESCRIPTION
**Pull request by awvwgk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by awvwgk** [comment]
@conda-forge/help-python This recipe is ready for review

File: recipes/data2objects/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "data2objects" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.8" %}
```
Comment (ocefpaf) [inline recipes/data2objects/meta.yaml:3]:
```suggestion
```

File: recipes/graph-pes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{% set name = "graph-pes" %}
+{% set version = "0.0.11" %}
+{% set python_min = "3.8" %}
```
Comment (ocefpaf) [inline recipes/graph-pes/meta.yaml:3]:
```suggestion
```

File: recipes/locache/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{% set name = "locache" %}
+{% set version = "4.0.1" %}
+{% set python_min = "3.8" %}
```
Comment (ocefpaf) [inline recipes/locache/meta.yaml:3]:
```suggestion
```

**Comment by awvwgk** [comment]
Since all recipes are noarch, passing on Linux CI seems sufficient.

**Comment by ocefpaf** [comment]
> Since all recipes are noarch, passing on Linux CI seems sufficient.

That is not always true. There are cases where we can build a broken noarch for Windows or macOS. We need to investigate the error. This one is odd:

```
    - python {{ python_min }}
jinja2.exceptions.UndefinedError: 'python_min' is undefined
```

B/c that should be defined in the CIs here for staged-recipes.

**Pull request merged**